src/service/gps_response.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package helium.mapper;
+
+message gps_response_t {
+  oneof version { gps_response_v1 gps_response_v1 = 1; }
+}
+
+message gps_response_v1 {
+  /**<   Unix time in seconds */
```

Comment:
Let's use the same `//` comment style throughout please

Reply:
You got it